modules/global/src/com/haulmont/addon/ldap/service/MatchingRuleAccessGroupService.java

```diff
@@ -0,0 +1,10 @@
+package com.haulmont.addon.ldap.service;
+
+import com.haulmont.addon.ldap.entity.AbstractDbStoredMatchingRule;
+import com.haulmont.cuba.security.entity.Group;
+
+public interface MatchingRuleAccessGroupService {
+	String NAME = "ldap_MatchingRuleAccessGroupService";
+
+	Group getAccessGroupMatchingRule(AbstractDbStoredMatchingRule rule);
```

Comment:
When you create a new service and a method in a service, it's welcomed to write a javadoc